examples/demo_bgp.pp

```diff
@@ -19,7 +19,29 @@
   # --------------------------------------------------------------------------#
   # Configure Global BGP                                                      #
   # --------------------------------------------------------------------------#
-  cisco_bgp { '55.77 blue':
+
+  $maxas_limit = $operatingsystem ? {
+    'nexus' => '50',
+    default => undef
```

Comment:
Yes, I believe so.  I've seen that no "UnsupportedError" is thrown when setting unsupported properties to "default" (at least some of the properties).